modules/virt-creating-bridge-nad-cli.adoc

```diff
@@ -14,7 +14,10 @@ As a network administrator, you can configure a network attachment definition of
 
 .Procedure
 
-. Create a network attachment definition manifest.
+. Create a network attachment definition manifest. Ensure that the network attachment definition uses the same namespace as the virtual machine.
```

Comment:
Nit: The network attachment definition is not using a namespace, it is created in a namespace.

Reply:
Changed to: Create a network attachment definition manifest. Ensure that you create the network attachment definition in the same namespace as the virtual machine.